core/src/main/java/org/apache/iceberg/CatalogProperties.java

```diff
@@ -30,6 +30,15 @@ private CatalogProperties() {
   public static final String FILE_IO_IMPL = "io-impl";
   public static final String WAREHOUSE_LOCATION = "warehouse";
 
+  public static final String TABLE_CACHE_ENABLED = "cache-enabled";
+  public static final boolean TABLE_CACHE_ENABLED_DEFAULT = true;
+  // If cache expiration interval is 0, we disable caching.
```

Comment:
nit: newline from the last variable, and can use `/**` for block comment instead of `//`

Reply:
Fixed.